### PR TITLE
✨ Change scripts/ to kubectl plugins

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates the scripts in scripts/ to become kubectl plugins according to the new command line design.  This PR also updates their documentation in scripts.md and the usage illustration in example1.md.

This is related to #350 and #347

## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/en/docs/Coding Milestones/PoC2023q1/scripts.md [hunk 1]
```diff
@@ -61,28 +70,39 @@ spec:
...
 $ kubectl ws root:imw-1
 Current workspace is "root:imw-1".
 
-$ bin/remove-location.sh demo1
+$ kubectl kcp-edge remove location demo1
 synctarget.workload.kcp.io "demo1" deleted
 location.scheduling.kcp.io "demo1" deleted
 
-$ bin/remove-location.sh demo1
+$ kubectl kcp-edge remove location demo1
```
Comment (waltforme) [inline docs/content/en/docs/Coding Milestones/PoC2023q1/scripts.md:105]: Is this line intentional (to show the script won't do anything unnecessary)?

Comment (MikeSpreitzer) [reply docs/content/en/docs/Coding Milestones/PoC2023q1/scripts.md:105]: Exactly.

Comment (waltforme) [reply docs/content/en/docs/Coding Milestones/PoC2023q1/scripts.md:105]: Cool.

**Comment by MikeSpreitzer** [comment]
The force-push to 5fc2914f completes a minor stylistic change to the kubectl command lines, moving the `--context` to appear as early as possible.

File: scripts/kubectl-kcp_edge-ensure-wmw [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+if [ "$want_kube" == true ] && ! kubectl "${kubectl_flags[@]}" get APIBinding bind-kube &> /dev/null; then
+kubectl "${kubectl_flags[@]}" apply -f - <<EOF
+apiVersion: apis.kcp.io/v1alpha1
+kind: APIBinding
+metadata:
+  name: bind-kube
+spec:
+  reference:
+    export:
+      path: "root:compute"
```
Comment (waltforme) [inline scripts/kubectl-kcp_edge-ensure-wmw:112]: Just one nit. Maybe we want to be consistent within the script, regarding whether use quotes or not (line 98).

Comment (MikeSpreitzer) [reply scripts/kubectl-kcp_edge-ensure-wmw]: Fixed in latest revision.

**Comment by MikeSpreitzer** [comment]
The force-push to aa06500c picks one nit noted in review, and does a squash.

**Pull request merged**